workflow/rules/sbs.smk

```diff
@@ -91,7 +91,7 @@ rule apply_ic_field_sbs:
         lambda wildcards: output_to_input(
             PREPROCESS_OUTPUTS["calculate_ic_sbs"],
             wildcards=wildcards,
-            subset_values={"cycle": SBS_CYCLES[config["sbs"]["dapi_index"]]},
+            subset_values={"cycle": 1},
```

Comment:
Good catch. Thought I fixed this in main but it seems I still had cycle 0 in main. You will have to resolve this merge conflict by accepting the change with cycle 1.